prov/verbs/src/ep_rdm/verbs_utils.c

```diff
@@ -197,8 +198,10 @@ int fi_ibv_rdm_find_ipoib_addr(const struct sockaddr_in *addr,
 
 	strncpy(iface, iface_tmp, iface_len);
 
-	getifaddrs(&addrs);
-	tmp = addrs;
+	if (!getifaddrs(&addrs)) {
+		tmp = addrs;
+	}
```

Comment:
You could goto line 220 if getifaddrs fails. This would also prevent freeing addrs in line 218 when it's not set.